learn/faq.md

```diff
@@ -54,6 +114,54 @@ The Python packaging system is prone to develop incompatibilities over time; the
 
+## Can I share conda environments with others? If so, how?
+
+Yes, you can share conda environments with others, making it easier to collaborate on projects or reproduce specific software setups. Here's how you can share conda environments:
+1. **Exporting the environment:** With conda, you can export the environment to a YAML file that contains a list of all the packages and their versions. Open your terminal or command prompt, activate the environment you want to share, and use the following command:
+```
+conda env export > environment.yml
+```
+This command exports the environment to an `environment.yml` file.
+
+2. **Sharing the environment file:** Once you have the `environment.yml` file, you can share it with others through various means, such as email, file sharing services, or version control systems like Git. Make sure to communicate any specific instructions or requirements related to the environment, such as additional dependencies or channels.
+
+3. **Creating the environment from the file:** To create an identical environment on another machine, the other person can use the following command in their terminal or command prompt:
+```
+conda env create -f environment.yml
+```
+This command reads the `environment.yml` file and recreates the environment with the same package versions and dependencies. The person needs to have conda or mamba installed on their system.
+Note: If using mamba instead of conda, the command will be:
+```
+mamba env create -f environment.yml
+```
+4. **Activating the shared environment:** After the environment is successfully created, the other person can activate it with:
+```
+conda activate <environment-name>
+```
+or
+```
+mamba activate <environment-name>
+```
+Replace `<environment-name>` with the name of the shared environment.
+By following these steps, you can easily share conda environments, ensuring consistent dependencies and versions.
+Remember that channels play a crucial role in environment replication. If the environment relies on packages from specific channels other than the defaults, the other person might need to add those channels using:
+```
+conda config --add channels <channel-name>
+```
+or 
+```
+mamba config --add channels <channel-name>
+``` 
+before creating the environment from the YAML file.
+
+## I want to create a conda package for my project. Where do I start?
+
+To create a conda package for your project, you can start by referring to the official documentation provided by [conda-build](https://github.com/conda/conda-build).
+
+For step-by-step guidance on setting up your package, defining dependencies, handling different platforms, and more, you can checkout the [documentation for conda-build](https://docs.conda.io/projects/conda-build/en/stable/user-guide/tutorials/building-conda-packages.html) .
```

Comment:
"check out" should be two words here

Reply:
And you've got a space between the last word and the period at the end of the sentence that shouldn't be there.